style.css

```diff
@@ -0,0 +1,352 @@
+    
+}
+.sidebar-items__title li{
+    list-style: none;
+    font-size: 1rem;
+    padding-left: .5rem;
+}
+
+.resume {
+    max-width: 75%;
+    margin: 1rem;
+    /* border: 1px solid green; */
+}
+
+.resume h2 {
+    margin-bottom: .5rem;
+    font-size: 2rem;
+    font-weight: bold;
+    /* border: 1px solid red; */
+}
+.resume-info__school dl{
+    font-size: 1.5rem;
+    padding-bottom: 1rem;
+    margin-bottom: 1rem;
+    border-bottom: 1px solid #9B9987;
+}
+.resume-info__school dt{
+    font-size: 1.25rem;
+    margin-left: .5rem;
+    /* border: 1px solid blue; */
+}
+.resume-info__school dd{
+    font-size: 1rem;
+    margin-left: 1rem;
+    /* border: 1px solid blue; */
+}
+
+.resume-info__work {
+    font-size: 1.5rem; 
+}
+.resume-info__title{
+    font-size: 1.25rem;
+    list-style: none;
+    margin-left: .5rem;
+}
+.resume-info__worklist {
+    font-size: 1rem;
+    margin-left: 2rem;
+    margin-bottom: 1rem;
+}
+
+/* ***********FOOTER CSS *********** */
+
+.footer {
+    /* display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-content: center; */ 
+    border-top: 1px solid #9B9987;
+    margin-top: 1rem;
+    padding: 1rem 1rem;
+    color: #9B9987;
+    font-size: .75rem;
+}
+.footer-contact {
+    display: flex;
+    justify-content: space-evenly;
+    flex-direction: row;
+}
+
+
+.footer-contact li {
+    list-style: none;
+}
+
+.footer-contact li:hover {
+    animation-name: shake;
+    animation-duration: 1s;
+}
+
+.footer-contact a{
+    color: #9B9987;
+    text-decoration: none;
+    font-size: 3rem;
+}
+
+.footer-contact a:hover{
+    color: #DBE9EE;
+    text-decoration: none;
+}
+.footer p {
+    text-align: center;
+    margin: 1rem 0;
+}
+
+
+/* Responsive for once it hit 768px */
+
+@media only screen and (min-width: 600px){
+    body {
+        max-width: 1300px;
+    }
+
+    .header {
+        justify-content: space-evenly;
+        height: 8rem;
+    }
+    .header h1 {
+        display: none;
+    }
+
+    .header-desk{
+        /* padding: 2.25rem 0 0 0; */
+        list-style: none;
+        display: block;
+        white-space: nowrap;
+        /* width: 100%; */
+        margin: 0 1rem;
+        font-size: 1.5rem;
+        text-align: center;
+        color: #475B63;
+    }
+
+    .main-nav {
+        font-size: 1.5rem;
+        /* margin: 1rem 3rem; */
+    }
+
+    .main-nav__item {
+        margin: 1rem 2rem;
+        /* margin-right: 1rem; */
+    }
+
+    .aboutMe {
+        display: flex;
+        justify-content: flex-start;
+    }
+
+   .aboutMe-info {
+        max-width: 50%;
+        padding: 0;
+        margin: 0;
+   }
+
+   .aboutMe-info img {
+       padding: 0;
+       max-height: 100%;
+   }
+   
+   .aboutMe-info p {
+       margin: 2rem 0 0 0;
+       font-size: 1.5rem;
+   }
+    .project {
+        flex-direction: row;
+    }
+
+    .project-info {
+        max-width: 50%;
+        padding: 0 1rem;
+    }
+}
+
+@keyframes shake {
```

Comment:
![](https://render.bitstrips.com/v2/cpanel/38a6faf8-807d-4ae3-8159-8d2659b88cec-bb69060c-eccd-4c08-a1d6-595d0b37616d-v1.png?transparent=1&palette=1)

I would love to hear the story behind this.